.github/workflows/ethash-build.yml

```diff
@@ -0,0 +1,43 @@
+name: Ethash
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+    types: [opened, reopened, synchronize]
+jobs:
+  lint:
```

Comment:
we can probably combine lint and coverage tasks into the single `test` job so it matches the other monorepo packages. we use `if:` if we don't need it to run multiple times across the matrix.

Reply:
good reminder! will normalize it.